src/AvaloniaEdit.Demo/MainWindow.xaml

```diff
@@ -23,7 +25,21 @@
             <Button Name="addControlBtn" Content="Add Button" VerticalAlignment="Stretch" VerticalContentAlignment="Center"/>
             <Button Name="clearControlBtn" Content="Clear Buttons" VerticalAlignment="Stretch" VerticalContentAlignment="Center"/>
             <ComboBox Name="syntaxModeCombo" VerticalAlignment="Stretch" VerticalContentAlignment="Center"/>
-            <Button Name="changeThemeBtn" Content="Change theme" VerticalAlignment="Stretch" VerticalContentAlignment="Center"/>
+            
+            <DropDownButton Content="Change theme" VerticalAlignment="Stretch" VerticalContentAlignment="Center">
+                <Button.Flyout>
+                    <Flyout>
+                        <ListBox ItemsSource="{Binding AllThemes }" SelectedItem="{Binding SelectedTheme}">
```

Comment:
It can be MenuFlyout with ToggleType=Radio, in 11.1 Avalonia.

Reply:
The toggle type radio is pretty awesome, I'm using it in Blitz Search but sure if that's something we want with this library? since 11.1 is all marked rc1,beta, etc.